pkg/controllers/vdb/verticadb_controller.go

```diff
@@ -258,6 +258,8 @@ func (r *VerticaDBReconciler) constructActors(log logr.Logger, vdb *vapi.Vertica
 		MakeClientRoutingLabelReconciler(r, log, vdb, pfacts, AddNodeApplyMethod, ""),
 		// Handle calls to add subclusters to sandboxes
 		MakeSandboxSubclusterReconciler(r, log, vdb, pfacts, dispatcher, r.Client, false),
+		// Handle calls to create archive
+		MakeCreateArchiveReconciler(r, vdb, log, pfacts, dispatcher, r.Client),
```

Comment:
Put this after line 269

Reply:
Updated in https://github.com/vertica/vertica-kubernetes/pull/918/commits/5663695a6b9c198cff6a6de9c01a62b6bc35ebbf